modules/s3db/hrm.py

```diff
@@ -5616,7 +5616,7 @@ def hrm_rheader(r, tabs=[],
         experience_tab = None
         service_record = ""
         tbl = TABLE(TR(TH(name,
-                          # @ToDo: Move to CSS
+                          _class="hrm-person-th",
                           _style="padding-top:15px")
```

Comment:
What's the point of this now ? ;)
Try these as well ? https://github.com/edengci/eden/pull/75/files#diff-9d3babe940b4fd8953ff3048869b308cR5689